projects/packages/forms/src/contact-form/class-util.php

```diff
@@ -287,6 +279,24 @@ public static function jetpack_tracks_record_grunion_pre_message_sent( $post_id,
 			'feedback_id'     => esc_attr( $all_values['feedback_id'] ),
 		);
 
+		// old deprecated attribute that should been removed in May 2020.
+		if (
+			! isset( $extra_values['is_block'] )
+			|| ! $extra_values['is_block']
+		) {
+			$event_props['is_block'] = 0;
+		} else {
+			$event_props['is_block'] = 1;
+		}
+
+		$post = get_post( $post_id );
+		if ( $post ) {
+			$event_props['post_w'] = gmdate( 'Y-W', strtotime( $post->post_date_gmt ) );
+			$event_props['post_m'] = gmdate( 'Y-m', strtotime( $post->post_date_gmt ) );
```

Comment:
Would there be a way to make a single date query and extract the week and the month from there, instead of calling `gmdate` twice? I wonder if that would make things easier to understand in the future.

Reply:
Coming back to this, and after discussion, this is probably fine. Other solutions would be overkill.

Related discussion: p1677689262478629-slack-C01CSBEN0QZ